expan/core/early_stopping.py

```diff
@@ -125,6 +126,37 @@ def HDI_from_MCMC(posterior_samples, credible_mass=0.95):
     return (HDImin, HDImax)
 
 
+def compile_stan_model(model_file, distribution):
+    """
+    Creates Stan model. Compiles a Stan model and saves it to .pkl file to the folder selected by tempfile module if
+        file doesn't exist yet and load precompiled model if there is a model file in temporary dir.
+    Args:
+        model_file: model file location
+        distribution: name of the KPI distribution model, which assumes a 
+            Stan model file with the same name exists
+    Returns:
+        returns compiled Stan model for the selected distribution or normal distribution
+            as a default option
+    Note: compiled_model_file is the hardcoded file path which may cause some issues in future.
+    There are 2 alternative implementations for Stan models handling:
+        1. Using global variables
+        2. Pre-compiling stan models and adding them as a part of expan project
+        (3). Using temporary files with tempfile module is not currently possible, since it 
+            generates a unique file name which is difficult to track.
+        However, compiled modules are saved in temporary directory using tempfile module 
+        which vary based on the current platform and settings. Cleaning up a temp dir is done on boot.
+    """
+    compiled_model_file = tempfile.gettempdir() + '/expan_early_stop_compiled_stan_model_' + distribution + '.pkl'
+
+    if os.path.isfile(compiled_model_file):
```

Comment:
Is there a way to ensure that `compiled_model_file` is the same file each time you call this function?